Sources/Sentry/SentryTracer.m

```diff
@@ -439,6 +438,108 @@ - (BOOL)hasChildrenToWaitFor
     }
 }
 
+#if SENTRY_HAS_UIKIT
```

Comment:
`h`: We are still missing tests for this code. I think you already mentioned that, but adding this comment not to forget.

Reply:
Sorry, I refactored this code into `SentryProfiler`. Would you mind pointing me to it again? I'm not sure of what I might have said before, but we do have tests for a couple combinations of scenarios of running the profiler now.